openlibrary/core/civicrm.py

```diff
@@ -25,7 +24,7 @@ def get_contact_id_by_username(username):
     data['json'] = json.dumps(data['json'])  # flatten the json field as a string
     r = requests.get(
         lending.config_ia_civicrm_api.get('url', ''),
-        params=urllib.urlencode(data),
```

Comment:
I think this is required because book titles have special characters.

Reply:
@mekarpeles `requests` handles url parameter encoding, it doesn't need to be a separate step